psst-gui/src/widget/icons.rs

```diff
@@ -94,6 +94,13 @@ pub static PLAY_LOOP_ALL: SvgIcon = SvgIcon {
     op: PaintOp::Fill,
 };
 
+// Font awesome - reload
```

Comment:
We should get all icons from SF Pro Regular https://developer.apple.com/sf-symbols/

Reply:
If you want, on Saturday I can go through and replace all of the icons with those ones?

Reply:
> If you want, I can go through and replace all of the icons with those ones?

Hold on, I am unable to access any of those symbols as I don't have an Apple machine!

Reply:
https://github.com/brendanballon/sfsymbols-svg/tree/master/symbols

I think the SVG's can be found at other places too!

Reply:
I think I will rewrite the way that the program processes the SVG files so then we are able to place all of the files into a directory rather than having to extract the curve. It would save a lot of time in the end!

Reply:
Hmm, I think part of the reason was that the SVG paths alone took up less size and might be quicker to load?

Reply:
Ok, do you know if there's an efficient way to move the "view" of the path as for example with the gear icon it doesn't start at the top left meaning that it's strongly displaced.

Reply:
@jacksongoode how does this look?

Reply:
These icons still look quite big from a UI consistency pov. I also think they need a little padding so they don't hit the right side too hard.